app/controllers/application_controller.rb

```diff
@@ -27,6 +28,10 @@ def identify_user!
     @current_user ||= (SessionService.identify_user!(session) || User.new)
   end
 
+  def set_sentry_user
+    Sentry.set_user(id: current_user.id)
```

Comment:
```suggestion
    Sentry.set_user(id: current_user&.id)
```

`SessionService.identify_user!` can return nil if the user is not signed in.

Reply:
A few lines above
```
@current_user ||= (SessionService.identify_user!(session) || User.new)
```
This should get rid of the need for a null/nil guard but i can add anyway. There's also a test for this to make sure it all works

Reply:
done